### PR TITLE
To Do list: List Structure

### PR DESCRIPTION
**Pull request by mmatongo**
### This pull request adds the following changes to the master branch of this repository.
- An `index.js` file that contains a simple array of tasks with three keys namely `description` `completed` `index`.
- A basic styles.css file that is loaded by webpack

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @mmatongo 👋

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+.list {
+  position: relative;
+  display: inline-block;
```
Comment (mikenath223) [inline src/styles.css:12]: - [ ] The project should look like the given design, kindly ensure it is a clone of the given design.

| Original | Yours |
| ----- | ------ |
|  |  |

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (vichuge) [inline package.json:8]: - [ ] I couldn't be able to run the project with `npm start`. Please, add the required to can do it. you can try adding `"start": "webpack serve --open"` after you'll need to install the webpack-dev-server plugin to continue. The same terminal helps you to install. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 2 above)
Comment (mmatongo) [reply package.json:8]: I'm confused, did I not meet all the project requirements?

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**